node_modules/oae-preview-processor/tests/test-previews.js

```diff
@@ -911,6 +929,23 @@ describe('Preview processor', function() {
         });
 
         /**
+         * Test that verifies blank thumbnails don't get added for unsupported mime types
+         */
+        it('verify default link processing does not attach blank thumbnails', function(callback) {
+            // Ignore this test if the PP is disabled.
+            if (!defaultConfig.previews.enabled) {
+                return callback();
+            }
+
+            _createContentAndWait('link', 'https://phs.googlecode.com/files/Download%20File%20Test.zip', null, function(restCtx, content) {
```

Comment:
this [link](https://phs.googlecode.com/files/Download%20File%20Test.zip) seems to be dead, so I guess the test is not valid?